hummingbot/connector/exchange/hotbit/hotbit_utils.py

```diff
@@ -0,0 +1,53 @@
+from decimal import Decimal
+from typing import Any, Dict
+
+from pydantic import Field, SecretStr
+
+from hummingbot.client.config.config_data_types import BaseConnectorConfigMap, ClientFieldData
+from hummingbot.core.data_type.trade_fee import TradeFeeSchema
+
+CENTRALIZED = True
+EXAMPLE_PAIR = "ZRX-ETH"
+
+DEFAULT_FEES = TradeFeeSchema(
+    maker_percent_fee_decimal=Decimal("0.001"),
+    taker_percent_fee_decimal=Decimal("0.001"),
+    buy_percent_fee_deducted_from_returns=True
+)
+
+
+class HotbitConfigMap(BaseConnectorConfigMap):
+    connector: str = Field(default="hotbit", const=True, client_data=None)
+    hotbit_api_key: SecretStr = Field(
+        default=...,
+        client_data=ClientFieldData(
+            prompt=lambda cm: "Enter your Hotbit API key",
+            is_secure=True,
+            is_connect_key=True,
+            prompt_on_new=True,
+        )
+    )
+    hotbit_api_secret: SecretStr = Field(
+        default=...,
+        client_data=ClientFieldData(
+            prompt=lambda cm: "Enter your Hotbit API secret",
+            is_secure=True,
+            is_connect_key=True,
+            prompt_on_new=True,
+        )
+    )
+
+    class Config:
+        title = "hotbit"
+
+
+KEYS = HotbitConfigMap.construct()
+
+
+def is_exchange_information_valid(exchange_info: Dict[str, Any]) -> bool:
```

Comment:
this method should be filtering the trading pairs that are not active or for example from perps

Reply:
The trading pairs returned from hobtit api do not include the pair from perps, and there is no property to distinguish whether it is active. I don't think they need to be filtered.